### PR TITLE
buttons and weapon change

### PR DESCRIPTION
**Pull request by kurebasu**
added some buttons and changed a weapon

### CODE REVIEW

File: Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab [hunk 1]
```diff
@@ -0,0 +1,1554 @@
+%YAML 1.1
```
Comment (mvanwoensel) [inline Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab:1]: Why did you make a copy of the enemy prefabs to your folder? 
I dont see a reason in your commit comments or summaries

File: Assets/02_Student Folders/SaidKrebbers_Assets/Scripts/compress.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  public Transform button;
+  public Transform weaponDoor;
+  public Collider triggerZone;
+
+
+
+  // Start is called before the first frame update
+  void Start()
+  {
+    pressing =false;
```
Comment (mvanwoensel) [inline Assets/02_Student Folders/SaidKrebbers_Assets/Scripts/compress.cs]: Unnecesarry if you are already doing that above

Comment (mvanwoensel) [reply Assets/02_Student Folders/SaidKrebbers_Assets/Scripts/compress.cs]: Doesn't have to be changed but just a small remark if you already gave it a value on creation you dont have to again assign it the same value on Start()

**Review by mvanwoensel — COMMENTED** [review]
Overall looks okay, I played the level didn't run in major bugs. 
I don't comment on overall quality/ looks of the level.

File: Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta [hunk 3]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```
Comment (mvanwoensel) [inline Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta]: Oh yeah why did you change/add stuff in somebody else his folder?

**Review by mvanwoensel — CHANGES_REQUESTED** [review]
Fix the changes you are doing in TayfunAygun's map or atleast explain why you are doing those

File: Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab (hunk 1 above)
Comment (kurebasu) [reply Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab:1]: I made copies of the enemies because I changed multiple things about them and did not want these changes to affect the others.

File: Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta (hunk 3 above)
Comment (kurebasu) [reply Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta]: This should not have happened, will fix

File: Assets/02_Student Folders/SaidKrebbers_Assets/Scripts/compress.cs (hunk 2 above)
Comment (kurebasu) [reply Assets/02_Student Folders/SaidKrebbers_Assets/Scripts/compress.cs]: That... is true, it is unnecessary (this statement was written during testing)
I do believe however that it is not necessarily wronk.

File: Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab (hunk 1 above)
Comment (kurebasu) [reply Assets/02_Student Folders/SaidKrebbers_Assets/Prefabs/Enemies/Enemy_Turret.prefab:1]: actually i just noticed i added half of the changes to the specific enemies in my lvls and not the prefabs themselves.
And some to the prefabs. which is slightly messy..., but oke for this assignment i believe

File: Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta (hunk 3 above)
Comment (kurebasu) [reply Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta]: I fixed the merge so that i will take what is in the master branch and not whatever i did in my branch.

**Comment by kurebasu** [comment]
I believe I fixed the changes sufficiently

**Review by mvanwoensel — APPROVED** [review]
lgtm
